crt_portal/cts_forms/views.py

```diff
@@ -1010,13 +1011,21 @@ def get_shared_report_values(self, record_query, keys):
         # make sure the queryset does not order by anything, otherwise
         # we will have difficulty getting distinct results.
         query = record_query.order_by()
+        query = query.annotate(
+            retention_year=F('retention_schedule__retention_years'),
+            expiration_year=F('retention_year') + ExtractYear('closed_date') + 1,
+            expiration_date=Cast(Concat(F('expiration_year'), Value('-'), Value('01'), Value('-'), Value('01'), output_field=CharField()), output_field=DateField())
```

Comment:
Nice use of the ORM to do this. Maybe a small cleanup here:

```suggestion
            expiration_date=Cast(Concat(F('expiration_year'), Value('-01-01'), output_field=CharField()), output_field=DateField())
```